src/graphics/components/movie/storyboard.cpp

```diff
@@ -52,11 +52,7 @@ MoveAlongPath::MoveAlongPath(float at, float duration, Path3 const& path,
     time_points_.push_back(at + duration);
 }
 
-void MoveAlongPath::update(float time) {
-    if (time < at_ || time > at_ + duration_) {
-        return;
-    }
-
+float MoveAlongPath::time_to_param(float time) {
```

Comment:
There should be a safeguard here, too, to make sure the loop for finding the indices really doesn't produce underflow.